src/main/java/org/wildfly/security/http/HttpServerRequest.java

```diff
@@ -63,6 +64,13 @@
     SSLSession getSSLSession();
 
     /**
+     * Get the peer certificates established on the connection.
+     *
+     * @return the peer certificates established on the connection or {@code null} if none available.
+     */
+    Certificate[] getPeerCertificates();
```

Comment:
Actually never mind, that's not really super important.  And who knows, maybe there'll be another certificate kind someday :)